src/zarr/abc/store.py

```diff
@@ -40,6 +60,17 @@ def from_literal(cls, mode: AccessModeLiteral) -> Self:
 
 
 class Store(ABC):
+    """
+    Abstract base class for Zarr stores.
+
+    Attributes
+    ----------
+    _mode : AccessMode
+        Access mode flags.
+    _is_open : bool
+        Whether the store is open.
```

Comment:
I don't think we should be documenting private attributes, so either the attributes docstring should go, or the attributes should become public.